app/models/tutorial.rb

```diff
@@ -20,14 +20,23 @@ def playlist_video_params
   end
 
   def video_params_values(video)
+    thumbnail = thumbnail_param(video)
     [video[:snippet][:title],
      video[:snippet][:description],
      video[:contentDetails][:videoId],
-     video[:snippet][:thumbnails][:default][:url],
+     thumbnail,
```

Comment:
if there is a `private video` than the thumbnail hash is empty. this will pass an empty string if it is a private video

Reply:
good catch